package.json

```diff
@@ -87,14 +87,14 @@
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
     "start": "webpack-dev-server",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "start-heroku": "terra express-server --port $PORT --site './build'",
+    "start-prod": "webpack-dev-server --env.disableHotReloading -p",
+    "start-static": "npm run compile:prod && terra express-server --site ./build",
     "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion"
+    "wdio-default": "terra wdio",
+    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 terra wdio --themes clinical-lowlight-theme",
+    "wdio-fusion": "terra wdio --themes orion-fusion-theme",
+    "wdio": "terra wdio --keepAliveSeleniumDockerService && NODE_OPTIONS=--max-old-space-size=2048 terra wdio --themes clinical-lowlight-theme --keepAliveSeleniumDockerService && terra wdio --themes orion-fusion-theme"
```

Comment:
i'm curious, with the theme context strategy, do we still need to increase the node memory via the node_options env variable?

Reply:
Oops, I didn't see this comment before merging. I'm not sure...this is to match the existing `wdio-lowlight` script.